_includes/header.html

```diff
@@ -1,44 +1,51 @@
 <!-- Navigation -->
-    <nav class="navbar navbar-default navbar-fixed-top">
```

Comment:
Le prime volte che ho usato Jekyll mi chiedevo anche io cosa fossero tutti questi spazi di indent nei file della cartella `_includes`: servono per avere i sorgenti della pagina "composta" tutti allineati. Non a caso la prima riga non e' mai identata.

Lo so, e' un dettaglio che possiamo ignorare ma in questo momento se guardiamo i sorgenti della pagina bolognatechscene.it ogni riga e identata coerentemente.